source/lib/rocprof-sys/library/rocprofiler-sdk.cpp

```diff
@@ -827,6 +827,121 @@ tool_tracing_buffered(rocprofiler_context_id_t /*context*/,
                                           _end_ns);
                 }
             }
+            else if(header->kind == ROCPROFILER_BUFFER_TRACING_HIP_RUNTIME_API)
+            {
+                auto* record = static_cast<rocprofiler_buffer_tracing_hip_api_record_t*>(
+                    header->payload);
+
+                if(record->operation == ROCPROFILER_HIP_RUNTIME_API_ID_hipEventRecord)
+                {
+                    auto _corr_id   = record->correlation_id.internal;
+                    auto _beg_ns    = record->start_timestamp;
+                    auto _end_ns    = record->end_timestamp;
+                    auto _thread_id = record->thread_id;
+                    auto _name      = tool_data->buffered_tracing_info.at(record->kind,
+                                                                     record->operation);
+
+                    if(get_use_timemory())
+                    {
+                        const auto& _tinfo =
+                            thread_info::get(record->thread_id, SystemTID);
+                        auto _tid = _tinfo->index_data->sequent_value;
+
+                        auto _bundle = kernel_dispatch_bundle_t{ _name };
+
+                        _bundle.push(_tid).start().stop();
+                        _bundle.get([_beg_ns, _end_ns](tim::component::wall_clock* _wc) {
+                            _wc->set_value(_end_ns - _beg_ns);
+                            _wc->set_accum(_end_ns - _beg_ns);
+                        });
+                        _bundle.pop();
+                    }
+
+                    if(get_use_perfetto())
+                    {
+                        auto _track_desc = [](rocprofiler_thread_id_t _tid) {
+                            const auto& _tid_v = thread_info::get(_tid, SystemTID);
+                            return JOIN("", "HIP Runtime API Thread ",
```

Comment:
```suggestion
                            return JOIN("", "GPU HIP Runtime Thread ",
```
Suggesting the "GPU" prefix, so it is grouped with the other domains.